### ISSUE TITLE
USA flag is wrong

### ISSUE THREAD

**Issue by Elardzhi** [SlavkoPekaric/Country-Flags-Responsive-CSS-Sprite#3]
is it a joke?

**Comment by excalith** [comment]
nope, it seems like it is intended for simplicity with the design in mind. lots of the flags are slightly off in their own way. for your case, adding 50 stars would look like a dot pattern within given resolution.

to implement this within this repository, lots of the flags should have alternate filenames with original flag design (which would set the flag design consistency of minimalism off - probably this is why you liked this icons at first) as seperate png's and add them into [this file](https://github.com/SlavkoPekaric/Country-Flags-Responsive-CSS-Sprite/blob/master/css/flags.css) so developers can choose which version to use. feel free to send pr if you do so, then no one thinks that open source is a joke! 😄

**Comment by jroovers** [comment]
^ Simplicity apparently is not important for all the other countries who have small stars on their flags, just USA.

It's now exactly like the flag of Liberia 😄, also looks like Puerto Rico.

Using these flags has been nice, and personally the USA flag doesn't irk me. But I too have now received comments from customers about the flag. I've had to manually replace the flag in a project I'm working on, but I'm not an artist, and you can tell from that the style does not match the other flags.
